static/js/mobile.js

```diff
@@ -0,0 +1,75 @@
+var loadingMessage = document.getElementById('msg');
+
+var useLoc = document.getElementById('use-loc');
+useLoc.checked = localStorage.useLoc === 'true';
```

Comment:
__[eslint]__
* Error - 'localStorage' is not defined. (no-undef)

<!-- Comment by [SideCI](https://sideci.com) -->